service/controller/v11/version_bundle.go

```diff
@@ -93,8 +118,8 @@ func VersionBundle() versionbundle.Bundle {
 		Dependencies: []versionbundle.Dependency{},
 		Deprecated:   false,
 		Name:         "kvm-operator",
-		Time:         time.Date(2018, time.April, 04, 17, 32, 0, 0, time.UTC),
+		Time:         time.Date(2018, time.April, 25, 13, 00, 0, 0, time.UTC),
```

Comment:
IMO we should not change the time. This was meant to be the time when the version bundle got created. I would like to have this reflecting the start of the version bundle lifecycle.

Reply:
where this time used? if customer sees it, then i would say we need to put a time, when bundle was released.

Reply:
This is used to calculate the release time. For me it is not important that we do it in one way or the other. For me it is important that we do it the same way based on what the team agreed on. So far we set the time once when the WIP bundle is created and it always stayed like this. When you change this now we get into a mess and I do not see a reason why we should do this. When we have consensus that we want it to do like this then I am fine with it but now you just changed it while all the other version bundles get treated differently. Lets please not create such exceptions.

Reply:
See for instance https://github.com/giantswarm/azure-operator/pull/190. The first Azure release is from January, even though we release it now.